components/catalogs/front/src/components/ProductSelection/criteria/AttributeYesNoCriterion/AttributeYesNoCriterion.integration.tsx

```diff
@@ -0,0 +1,390 @@
+const channelPrint = {code: 'print', label: 'Print'};
+
+const changeOperatorTo = (operator: string) => changeSelectValueTo('operator', operator);
+const changeChannelTo = (channel: string) => changeSelectValueTo('scope', channel);
+const changeLocaleTo = (locale: string) => changeSelectValueTo('locale', locale);
+const changeInputTo = (value: string) => changeSelectValueTo('value', value);
+const changeSelectValueTo = (selector: string, value: string) => {
+    const select = screen.getByTestId(selector);
+    fireEvent.click(within(select).getByRole('textbox'));
+    fireEvent.click(screen.getByText(value));
+};
+
+beforeEach(() => {
+    fetchMock.mockResponse(req => {
+        switch (req.url) {
+            // useAttribute
+            case '/rest/catalogs/attributes/name_localizable_scopable':
+                return Promise.resolve(
+                    JSON.stringify({
+                        label: 'Name',
+                        code: 'name',
+                        type: 'pim_catalog_boolean',
+                        scopable: true,
+                        localizable: true,
+                    })
+                );
+            case '/rest/catalogs/attributes/name_scopable':
+                return Promise.resolve(
+                    JSON.stringify({
+                        label: 'Name',
+                        code: 'name',
+                        type: 'pim_catalog_boolean',
+                        scopable: true,
+                        localizable: false,
+                    })
+                );
+            case '/rest/catalogs/attributes/name_localizable':
+                return Promise.resolve(
+                    JSON.stringify({
+                        label: 'Name',
+                        code: 'name',
+                        type: 'pim_catalog_boolean',
+                        scopable: false,
+                        localizable: true,
+                    })
+                );
+            // useChannel
+            case '/rest/catalogs/channels/ecommerce':
+                return Promise.resolve(JSON.stringify(channelEcommerce));
+            case '/rest/catalogs/channels/print':
+                return Promise.resolve(JSON.stringify(channelPrint));
+            // useChannelLocales
+            case '/rest/catalogs/channels/ecommerce/locales':
+                return Promise.resolve(JSON.stringify([localeUS, localeFR]));
+            // useLocales
+            case '/rest/catalogs/locales':
+                return Promise.resolve(JSON.stringify([localeUS, localeFR, localeDE]));
+            // useInfiniteChannels
+            case '/rest/catalogs/channels?page=1&limit=20':
+                return Promise.resolve(JSON.stringify([channelEcommerce, channelPrint]));
+            default:
+                throw Error(req.url);
+        }
+    });
+});
+
+test('it renders the scopable and localizable text attribute criterion', async () => {
```

Comment:
```suggestion
test('it renders the scopable and localizable yes/no attribute criterion', async () => {
```

Also, should we call it boolean instead of yes/no ?
Internally, it's `pim_catalog_boolean` :thinking:

Reply:
Good question, it's the same things for the AttribueMeasurementCriterion with `pim_catalog_metric`.

Reply:
Ah ah, it's the other way around for the other. `metric` is a typo, It should have been called measurement from the start.
it is now, the code was refactored to be moved to MeasurementBundle. Only the attribute type was never renamed because of the impact.

Reply:
It raise an interesting point.
We need to update the Ubiquitous Language of the Catalogs feature.

Reply:
Renaming done :heavy_check_mark: => AttributeBooleanCriterion